mesheryctl/cmd/perf.go

```diff
@@ -86,7 +91,18 @@ var perfCmd = &cobra.Command{
 			log.Print(perfDetails)
 			return
 		}
+		if filePath != "" {
+			var t models.PerformanceSpec
+			err := yaml.Unmarshal([]byte(filePath), &t)
 
+			if err != nil {
+				log.Errorf("Error: File Supplied is invalid.\n%v", err)
```

Comment:
Question: Would "Error: The file provided contains invalid yaml." be a more descriptive (accurate) error to provide the user?

Reply:
Yeah, but the error might be occurring if the file was not present there or there were no permissions present... Exact reason of error will be printed in the next line... should we write "Invalid yaml file" instead? or keep it as is?

Reply:
Oh, I see. Yes,  "Invalid yaml file" sounds good, if we leave this to only one check. If we were to break into two checks - one for presence of a file and the other for validity of yaml - we could have two errors. I don't know that it's necessary to breakout into two checks. "Invalid yaml file" does cover them both.